.github/workflows/tests.yaml

```diff
@@ -77,11 +77,14 @@ jobs:
 
       - name: Install dependencies
         run: |
+          echo "::group::snap install"
+          sudo snap install chisel --edge
+          echo "::endgroup::"
           echo "::group::apt-get update"
           sudo apt-get update
           echo "::endgroup::"
           echo "::group::apt-get install..."
-          sudo apt-get install -y python3 python3-dev libapt-pkg-dev libyaml-dev umoci
+          sudo apt-get install -y python3 python3-dev libapt-pkg-dev libyaml-dev umoci git openjdk-21-jdk
```

Comment:
Why is openjdk needed since everything related to jdk seems to use chisel ?

Reply:
OpenJDK is needed on host to create jlink image. 
Chisel is used to install minimal set of dependencies for openjdk.